mmv1/products/integrationconnectors/Connection.yaml

```diff
@@ -0,0 +1,1120 @@
+          Additional SSL related field values.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "key"
+              description: |
+                Key for the configVariable
+              required: true
+            - !ruby/object:Api::Type::Integer
+              name: "integerValue"
+              api_name: "intValue"
+              description: |
+                Integer Value of configVariable.
+            - !ruby/object:Api::Type::String
+              name: "stringValue"
+              description: |
+                String Value of configVariabley.
+            - !ruby/object:Api::Type::Boolean
+              name: "booleanValue"
+              api_name: "boolValue"
+              description: |
+                Boolean Value of configVariable.
+            - !ruby/object:Api::Type::NestedObject
+              name: "secretValue"
+              description: |
+                Secret value of configVariable
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: "secretVersion"
+                  required: true
+                  description: |
+                    Secret version of Secret Value for Config variable.
+            - !ruby/object:Api::Type::NestedObject
+              name: "encryptionKeyValue"
+              description: |
+                Encription key value of configVariable
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: "type"
+                  description: |
+                    Type of Encription Key
+                  values:
+                    - GOOGLE_MANAGED
+                    - CUSTOMER_MANAGED
+                - !ruby/object:Api::Type::String
+                  name: "kmsKeyName"
+                  description: |
+                    The [KMS key name] with which the content of the Operation is encrypted. The expected
+                    format: projects/*/locations/*/keyRings/*/cryptoKeys/*.
+                    Will be empty string if google managed.
+  - !ruby/object:Api::Type::Enum
+    name: "subscriptionType"
+    description: |
+      This subscription type enum states the subscription type of the project.
+    output: true
+    values:
+      - SUBSCRIPTION_TYPE_UNSPECIFIED
+      - PAY_G
+      - PAID
+  - !ruby/object:Api::Type::String
+    name: "connectionRevision"
+    description: |
+      Connection revision. This field is only updated when the connection is created or updated by User.
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: "eventingEnablementType"
+    description: |
+      Eventing enablement type. Will be nil if eventing is not enabled.
+    values:
+      - EVENTING_AND_CONNECTION
+      - ONLY_EVENTING
+  - !ruby/object:Api::Type::NestedObject
+    name: "eventingConfig"
+    description: |
+      Eventing Configuration of a connection
+    properties:
```

Comment:
one of these required, or have at_least_one_of (and also for all the children fields)

Reply:
Done